handler/pagination.go

```diff
@@ -0,0 +1,30 @@
+package handler
```

Comment:
There are changes from your other branch here

Reply:
I would ignore this as if we merge the other PR in first then itll fold in nicely